<launch>
        <arg name="foo" default="False"/>
        <group if="$(arg foo)">
            <include file="$(find yolov4_trt_ros)/launch/video_source.launch" />
        </group>
	<node pkg="yolov4_trt_ros" type="demo_view" name="demo_view" output="screen">
            <param name="topic_name" value="/video_source/raw"/>
	</node>

</launch>
